app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<%# Regular Form %>
+
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
+  <label for="name">Username:</label>
+
+  <input type="text" id="name" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type = "hidden" name = "authenticity_token" value = "<%=form_authenticity_token%>">
+  <input type="submit" value="Submit">
+</form>
+-->
+
+
+<%# Form_tag %>
+
+<!--
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:name, "username") %> 
+  <%= text_field_tag('user[username]') %> <br><br>
+
+  <%= label_tag(:email,"email") %>
+  <%= text_field_tag('user[email]') %> <br><br>
+
+  <%= label_tag(:password, "password") %>
+  <%= text_field_tag("user[password]") %> <br><br>
```

Comment:
It must be a **password** field instead of a _textfield_
Fix that please

Reply:
Thanks, done!